mava/systems/jax/parameter_server.py

```diff
@@ -30,7 +30,15 @@ def __init__(
         store: SimpleNamespace,
         components: List[Callback],
     ) -> None:
-        """Initialise the parameter server."""
+        """Initialise the parameter server.
+
+        Args:
+            store: builder store.
+            components: components in the system.
+
+        Returns:
+            None.
```

Comment:
The return type doesn't seem to be added for the inits in the other classes. Is there a specific reason for the parameter server?

Reply:
Nice catch, thanks! Removed this one.
I've been leaving out return types for init methods because they should always return `None`.

Reply:
Thanks for the clarification.